homeassistant/helpers/entity.py

```diff
@@ -631,12 +631,12 @@ def entity_registry_visible_default(self) -> bool:
             return self.entity_description.entity_registry_visible_default
         return True
 
-    @property
+    @cached_property
```

Comment:
What would be the gain in this case?

Reply:
It would only make a difference if its subclassed and the subclass has more complex logic